pkg/morph/client/neofs/client.go

```diff
@@ -28,7 +28,7 @@ const (
 )
 
 // NewFromMorph wraps client to work with NeoFS contract.
-func NewFromMorph(cli *client.Client, contract util.Uint160, fee fixedn.Fixed8, opts ...Option) (*Client, error) {
+func NewFromMorph(cli *client.Client, contract util.Uint160, opts ...Option) (*Client, error) {
```

Comment:
I think this can be kept as is, just handle client.WithStaticFeeIncrement internally.